taskcluster/app_services_taskgraph/__init__.py

```diff
@@ -33,6 +32,8 @@ def register(graph_config):
         # "nightly" for the nightly builds.  Other strings indicate making a
         # preview build for a particular application-services branch.
         'preview-build': str,
+        # Release type.  We set this to "release" for non-nightly builds, it defaults to "nightly".
+        'release': Optional(str),
```

Comment:
Gecko calls this parameter `release-type`, might be worth making it consistent.

Reply:
Sounds good, I updated the param.